client/my-sites/sidebar/utils.js

```diff
@@ -41,7 +41,7 @@ export const itemLinkMatches = ( path, currentPath ) => {
 
 	if ( pathIncludes( currentPath, 'settings', 1 ) ) {
 		// Jetpack Cloud uses a simpler /settings/:site pattern for the settings page.
```

Comment:
Nitpicking: Probably update the comment here also to mention A4A.

Reply:
I saw it, but that comment is specific to the pattern `/settings/:site`, which we don't have at A4A.

Reply:
`// Jetpack Cloud uses a simpler /settings/:site pattern, and A4A /settings/:tabs, for the settings page.`

At A4A we will have tabs, so that we will have that pattern: `/settings/:tabs`